content/zh/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -63,46 +63,84 @@ Pilot 根据用户编写的配置文件和系统当前状态来配置 sidecar 
+A large number of listeners, clusters, and routes can increase memory usage.
+In a large namespace with [namespace isolation](/docs/reference/config/networking/sidecar/) enabled, the proxy consumes approximately 50 MB of memory.
+-->
+代理的内存消耗取决于它的总体配置状态。大量的监听器、集群和路由会增加内存使用量。在启用了[命名空间隔离](/zh/docs/reference/config/networking/sidecar/)的大型命名空间中，代理消耗大约 50 MB 的内存。
 
 由于代理通常不缓存通过的数据，请求速率不会影响内存消耗。
 
 ### 延迟 {#latency}
 
-因为 Istio 在数据路径上注入了一个 sidecar 代理，所以延迟是重要的考量因素。Istio 向代理添加了身份验证和 Mixer 过滤器。每一个额外的过滤器都会增加代理内的路径长度并影响延迟。
-
-响应被返回给客户端后，Envoy 代理将收集原始的遥测数据。为请求收集原始遥测数据所花费的时间并没有统计在完成该请求所需的总时间里。但是，worker 在忙着处理请求时是不会立刻开始处理下一个请求的。此过程会增加下一个请求的队列等待时间并影响平均延迟和尾部延迟。实际的尾部延迟取决于流量模式。
-
-在网格内部，一个请求会先遍历客户端代理，然后遍历服务端代理。在 90% 的情况下，数据路径上的这两个代理每 1000 QPS 会产生 6.3 ms 的延迟。
-
-90% 的情况下仅服务端代理会增加 1.7 ms 的延迟。
+<!--
+Since Istio injects a sidecar proxy on the data path, latency is an important consideration. Every feature Istio adds also adds to the path length inside the proxy and potentially affects latency.
+-->
+由于 Istio 在数据路径上注入了一个 sidecar 代理，因此延迟是一个重要的考虑因素。 Istio 添加的每个功能也会增加代理内部的路径长度，并可能影响延迟。
+
+<!--
+The Envoy proxy collects raw telemetry data after a response is sent to the
+client. The time spent collecting raw telemetry for a request does not contribute
+to the total time taken to complete that request. However, since the worker
+is busy handling the request, the worker won't start handling the next request
+immediately. This process adds to the queue wait time of the next request and affects
+average and tail latencies. The actual tail latency depends on the traffic pattern.
+-->
+Envoy 代理在向客户端发送响应后收集原始遥测数据。为请求收集原始遥测数据所花费的时间不计入完成该请求所花费的总时间。但由于 worker 正忙于处理请求，它不会立即开始下一个请求的处理。此过程会增加下一个请求的队列等待时间，并影响平均延迟和尾延迟。实际的尾部延迟取决于流量模式。
 
 ### Istio {{< istio_release_name >}} 的延迟 {#latency-for-Istio}
 
-Istio {{< istio_release_name >}} 的默认配置在 90% 的情况下使数据平面的延迟比基线增加了 6.3 ms。我们通过 `http/1.1` 协议的 [Istio 基准测试](https://github.com/istio/tools/tree/master/perf/benchmark)获得了结果，测试标准是每秒 1000 请求，负载为 1KB，使用了 16 个客户端连接和 2 个代理，双向 TLS 打开状态。
+<!--
+Inside the mesh, a request traverses the client-side proxy and then the server-side
+proxy. In the default configuration of Istio {{< istio_release_name >}} (i.e. Istio with telemetry v2),
+the two proxies add about 1.7 ms and 2.7 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
+We obtained these results using the [Istio benchmarks](https://github.com/istio/tools/tree/{{< source_branch_name >}}/perf/benchmark)
+for the `http/1.1` protocol, with a 1 kB payload at 1000 requests per second using 16 client connections, 2 proxy workers and mutual TLS enabled.
+-->
+在 mesh 内部，请求会依次遍历客户端和服务器端代理。在 Istio {{< istio_release_name >}} 的默认配置中（即带有遥测 v2 的 Istio），
+两个代理分别在基线数据平面延迟的 90 和 99 分位延迟上增加约 1.7 和 2.7 毫秒。
+我们使用 `http/1.1` 协议的 [Istio 基准测试](https://github.com/istio/tools/tree/{{< source_branch_name >}}/perf/benchmark)获得了这些结果，
+测试标准是每秒 1000 请求，负载为 1KB，使用了 16 个客户端连接和 2 个代理 wroker 并启用双向 TLS 。
```

Comment:
wroker to worker :)

Reply:
I found typo `worker -> worker`, the number `2` is already plural, I don't think `workers` need to be used, thx~